### ISSUE TITLE
Fails to build and run for iOS

### ISSUE THREAD

**Issue by okko-darren** [Glartek/flutter-unity#7]
**Describe the bug**

Fails to build and run for iOS with following output:

```
Launching lib/main.dart on iPhone in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: ZP4N569NB5
Xcode build done.                                            5.4s
Failed to build iOS app
Error output from Xcode build:
↳
    2020-08-12 17:18:24.534 xcodebuild[34553:236382]  DTDeviceKit: deviceType from 38ad642af5b1c27878b59e12ad50adb0ce78c4f4 was NULL
    ** BUILD FAILED **
Xcode's output:
↳
    /Users/tmw/.pub-cache/hosted/pub.dartlang.org/flutter_unity-0.2.1+4/ios/Classes/FlutterUnityPlugin.m:3:9: fatal error: 'UnityFramework/UnityFramework.h' file not found
    #import <UnityFramework/UnityFramework.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
Could not build the precompiled application for the device.

Error launching application on iPhone.
Exited (sigterm)
```

**To Reproduce**

1. Start a new flutter project - currently this specifies `environment: sdk: ">=2.7.0 <3.0.0"`
2. Add `flutter_unity:` to `dependencies:` as per pub.dev installation instructions
3. Build the example Unity project from this repo
4. Follow all 6 steps here: https://github.com/Glartek/flutter-unity#ios-2
5. Run 

**Expected behavior**

Successful compilation and a default project loaded on iPhone

**Additional context**

Thanks for the awesome work on this, building fine on windows for android but can't get iOS to play ball.

I can successfully build and run the example project but cannot start a new project or add `flutter_unity` to an existing project successfully.

Mac OS: 10.15.5
Unity: 2019.3.7f1
XCode: 11.5
Visual Studio Code: 1.47.3

**Comment by cookiejarlid** [comment]
From what I gather, you were able to successfully build using the example project, but not when starting a new flutter project. Is that correct?

Xcode can be a "difficult" tool to use, and the steps for iOS can be a bit hard to follow. With that said, I've just tested creating a new flutter project with the latest stable flutter version (currently 1.20.2) and embedding the included unity example project and I couldn't find any issues. Could you please try again from scratch and let me know?

**Comment by okko-darren** [comment]
Thanks for your reply, I've just tried again, no difference. 

Some screenshots of the different parts of your successful XCode project might be useful, parts where settings are changed or just if there is any small additional step that's maybe taken for granted but not explicitly explained in the iOS steps on the Readme?

**Comment by okko-darren** [comment]
I've realised I was opening `Runner.xcodeproj` instead of `Runner.xcworkspace` - I assume this is where my problem is!! Bare with me and I'll close the issue if it's resolved.

**Comment by okko-darren** [comment]
I'm all good now 🤦‍♂️ Silly mistake, thanks for you patient reply!
